sites/docs/src/content/docs/tutorials/switching_barnch_to_main/switching_master_to_main.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Transitioning from `master` to `main` branches in pipelines"
+subtitle: Guide on how to switch a pipeline from the default branch `master` to `main`
+weight: 20
+---
+
+This tutorial will guide you through changing the pipeline default branch form `master` to `main`.
+
+## Prequisites
+
+- Admin rights on the GitHub repository where the change taking place
+- nf-core tools
+  - It is recommended that the version match the version of the template in your pipeline
+
+> [!WARNING]
+> Make sure you are on using same version of nf-core/tools as the template version of your branch.
+> This will avoid adding other changes not needed for the branch name switch.
+
+## Instructions
+
+1. Open your repository on GitHub (make sure you're on master)
+2. Go to the `branches` page (to the right of the `branches` drop down where it says 'master')
+
+![image](https://hackmd.io/_uploads/rkc8GXJWgx.png)
```

Comment:
please add these images to the PR inside `/public/images/tutorials`

Reply:
done. But I don't see then in the preview, did i do something wrong?